pygeoapi/api/__init__.py

```diff
@@ -1425,6 +1425,8 @@ def get_collection_schema(api: API, request: Union[APIRequest, Any],
 
     for k, v in p.fields.items():
         schema['properties'][k] = v
+        if v['type'] == 'float':
```

Comment:
Maybe add a comment on why we are setting the schema different than the value returned from the provider.